artemis-server/pom.xml

```diff
@@ -53,6 +53,11 @@
       </dependency>
       <dependency>
          <groupId>org.apache.activemq</groupId>
+         <artifactId>artemis-boot</artifactId>
```

Comment:
remove this?


the boot is just a classloader thing.. no component should ever need to look into this.

Reply:
removed